force-app/main/classes/FFAuthServiceCached.cls

```diff
@@ -60,12 +74,16 @@ public class FFAuthServiceCached implements FFAuthService {
             FFModels.JWTClaims claims = this.claimsFrom(result.authToken);
             FFModels.AuthInfo authClaims = new FFModels.AuthInfo(result.authToken, claims.environment, claims.clusterIdentifier);
             this.updateAuthInCache(authClaims);
+            Long authTime = Datetime.now().getTime();
+            this.updateLastAuthTimeInCache(authTime);
             System.debug('Env UUID ' + authClaims.environmentUUID);
             System.debug('Cluster ID ' + authClaims.cluster);
+            System.debug('Auth time ' + authTime);
             return authClaims;
         } catch (FFBase.ApiException e) {
             System.debug('Auth Error');
             System.debug(e);
+            // TODO retry on more than just 403
```

Comment:
will this need to be addressed?

Reply:
Cheers.. yes, I'll add a 500 to the list now

Reply:
Done, and added an additional unit test.